external/tfmicro/tensorflow/lite/core/api/error_reporter.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is there a need for Samsung electronics License addition in external files? Kindly check

Reply:
@Taejun-Kwon What is your opinion? There are many external files.